portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -1,16 +1,3 @@
-// Copyright 2019 Google LLC
```

Comment:
I'm curious why you're deleting the copyright lines?

Reply:
I didn't even notice that I did that